Dshop/apps/users/tests/test_api_register.py

```diff
@@ -0,0 +1,83 @@
+import pytest
+from django.contrib.auth import get_user_model
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APIClient
+
+from ..models import CustomUser
+
+User = get_user_model()
+
+
+@pytest.fixture
+def api_client():
+    return APIClient()
+
+
+@pytest.fixture
+def registration_url():
+    return reverse('api-register')
+
+
+@pytest.fixture
+def registration_data():
+    return {
+        'username': 'testuser',
+        'email': 'testuser@example.com',
+        'password': 'testpassword',
+        'password_again': 'testpassword',
+    }
+
+
+@pytest.mark.django_db
+def test_registration_success(api_client, registration_url, registration_data):
+    response = api_client.post(registration_url, registration_data, format='json')
+    assert response.status_code == status.HTTP_201_CREATED

```

Comment:
As with the previous one, please also add some tests for response data - bit more than just response code.